src/VisualStudio/Core/Def/project.json

```diff
@@ -18,6 +18,7 @@
     "Microsoft.VisualStudio.Editor": "15.0.26201-alpha",
     "Microsoft.VisualStudio.Language.StandardClassification": "15.0.26201-alpha",
     "Microsoft.VisualStudio.Language.Intellisense": "15.0.26201-alpha",
+    "Microsoft.VisualStudio.Sdk.TaskStatusCenter": "15.0.212-pre",
```

Comment:
Is it safe to reference this in VS2017 RTW?  Do we need to do lightup work for it?